source/wp-content/themes/wporg-news-2021/sass/page/front-page/_people-of-wordpress.scss

```
@@ -72,9 +56,35 @@ body.news-front-page .front__people-of-wordpress {
 			}
 
 			&.has-post-thumbnail {
-				.wp-block-post-title {
+				.wp-block-post-featured-image {
+					margin: 0;
+
+					img {
+						object-fit: cover;
+						filter: grayscale(100%);
+						aspect-ratio: 1 / 1;
+					}
+				}
+				
+				&:hover,
+				&:focus-within {
+					figure {
+						transition: all 0.3s ease-in-out;
+						background-color: var(--wp--preset--color--blue-1);
+					}
+
+					img {
+						mix-blend-mode: multiply;
+					}
+
+					.wp-block-post-title {
```

I noticed this stutters a bit when tabbing through the content, I'm not entirely sure _why_, but putting the opacity & transition on the `a` tag instead seems to help.

```suggestion
					.wp-block-post-title a {
```